src/plugins/graph/components/graph-axis.tsx

```diff
@@ -73,14 +73,14 @@ export const GraphAxis = observer(function GraphAxis({
           const axisModel = graphModel.getAxis(place);
 
           if (axisModel && isNumericAxisModel(axisModel)) {
-            if (xValues && place === "bottom") {
+            if (xValues?.length > 0 && place === "bottom") {
```

Comment:
```suggestion
            if (xValues.length > 0 && place === "bottom") {
```
`numericValuesForAttrRole` is guaranteed to return an array, so there's no need for the optional chain. Furthermore, if it were possible for `xValues` to be `undefined`, then TypeScript would complain because `undefined` can't be order-compared with `number`.